go.mod

```diff
@@ -5,14 +5,17 @@ go 1.19
 require (
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.27.7
+	github.com/redhat-appstudio/application-api/model v0.0.0-00010101000000-000000000000
```

Comment:
Trying to consume the changes here in HAS gives me the following error:
```
appstudio/application-api/model@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```

Would changing version here to something like `v0.0.0` help?

Reply:
Actually, changing it to `v0.0.0` gives me a different error:
```
	github.com/redhat-appstudio/application-api/model: reading github.com/redhat-appstudio/application-api/model/go.mod at revision model/v0.0.0: unknown revision model/v0.0.0
```

Reply:
There is no `github.com/redhat-appstudio/application-api/model`, well not unless this PR is merged.

This should work in a `go.mod`:

```
replace github.com/redhat-appstudio/application-api/model => github.com/zregvart/application-api/model v0.0.0-20230731114935-99992a8e0580

require github.com/redhat-appstudio/application-api/model v0.0.0-00010101000000-000000000000
```

Reply:
I think this may also work

```
replace github.com/redhat-appstudio/application-api/model => ./model
```

Reply:
yes, so the following should work:

```
github.com/redhat-appstudio/application-api/model  v0.0.0


replace github.com/redhat-appstudio/application-api/model => ./model
```